mreg_cli/api/models.py

```diff
@@ -1281,7 +1281,7 @@ def output_atoms(self, _padding: int = 14) -> None:
             manager.add_line("No atom members")
 
     @classmethod
-    def output_multiple(cls, roles: list[Role], padding: int = 14) -> None:
+    def output_multiple(cls, roles: list[Role] | list[str], padding: int = 14) -> None:
```

Comment:
It seemed silly _not_ to support `list[Role]` here, but this method is now more limited due to also having to accept lists of role names. Could consider refactoring into 2 methods, one for each type, but deemed it not necessary here (yet).